.github/workflows/IJ.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        IJ: [IC-2021.1, IC-2021.2, IC-2021.3, IC-2022.1, IC-2022.2, IC-2022.3, IC-2023.1, IC-2023.2]
+        IJ: [IC-2022.3, IC-2023.1, IC-2023.2, IC-2024.1]
```

Comment:
means all dependent extensions using next version need to  validate this range version too ?

Reply:
I'd think so, yes. Agree?

Reply:
* intellij-kubernetes is at `IC-2022.1` (https://github.com/redhat-developer/intellij-kubernetes/blob/1f4637041eb0a0b36fb86a6589f1370f9406d702/.github/workflows/IJ.yml#L17)
* intellij-openshift-connector is at `IC-2022.1` (https://github.com/redhat-developer/intellij-openshift-connector/blob/30e4bc351d8df6a8058cf0d443706ab48cd2fed5/.github/workflows/validate_IJ_versions.yml#L12)
* intellij-tekton is at `IC-2022.2` (https://github.com/redhat-developer/intellij-tekton/blob/84f3d264c06b98290bcf88626d86fa8f5a085dd7/.github/workflows/IJ.yml#L17)
* intellij-knative is at `IC-2022.1` (https://github.com/redhat-developer/intellij-knative/blob/2e3c49bd684f27383a049c740e96f5e28a4f8ea4/.github/workflows/IJ.yml#L17)

Reply:
agree to move up to at least 22.3 faor all downstream. not a big step.